test/default_manifold.jl

```diff
@@ -636,11 +636,12 @@ using ManifoldsBaseTestUtils
         MC = ManifoldsBase.DefaultManifold(3; field = ManifoldsBase.ℂ)
         p = [1.0im, 2.0im, -1.0im]
         CB = get_basis(MC, p, DefaultOrthonormalBasis(ManifoldsBase.ℂ))
+        @test CB.data == [[1.0, 0.0, 0.0], [0.0, 1.0, 0.0], [0.0, 0.0, 1.0]]
         @test CB.data isa Vector{Vector{ComplexF64}}
```

Comment:
Is this test then still correct? For complex coefficients upfront, the eltype of the basis should be real (the `==` test above is even vs real basis vectors)

Reply:
At the moment `get_basis` returns vectors with complex eltype. It makes some sense from the point of view of type stability when operations are performed on those vectors but since all imaginary parts are 0, they could be real vectors.

At the moment it works like this because `p` has complex eltype and the code for producing basis vectors takes the eltype from `p`. Would you prefer to change it?

Reply:
I've found a reasonable way to address this issue I think.

Reply:
Oh, a very fancy fix, nice.